src/main/java/ch/njol/skript/expressions/ExprBrewingFuelLevel.java

```diff
@@ -0,0 +1,76 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.Math2;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Block;
+import org.bukkit.block.BrewingStand;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Brewing Stand Fuel Level")
+@Description("The fuel level of a brewing stand. The fuel level is decreased by one for each potion that is being brewed.")
```

Comment:
you should specify when it is decreased, e.g.

```suggestion
@Description("The fuel level of a brewing stand. The fuel level is decreased by one at the start of brewing each potion.")
```